packages/web-app-files/src/App.vue

```diff
@@ -55,13 +55,46 @@ export default defineComponent({
     })
   },
 
+  mounted() {
+    document.addEventListener('keydown', this.handleShortcut, false)
+  },
+
+  beforeDestroy() {
+    document.removeEventListener('keydown', this.handleShortcut)
+  },
+
   methods: {
     ...mapActions('Files', ['resetFileSelection']),
     ...mapActions('Files/sidebar', {
       closeSidebar: 'close',
       setActiveSidebarPanel: 'setActivePanel'
     }),
-    ...mapActions(['showMessage']),
+    ...mapActions(['showMessage', 'createModal', 'hideModal']),
+    ...mapActions('Files', ['copySelectedFiles', 'cutSelectedFiles', 'pasteSelectedFiles']),
+    ...mapMutations('Files', ['UPSERT_RESOURCE']),
+
+    handleShortcut(event) {
+      console.log(event)
+      const key = event.keyCode || event.which
+      const ctr = window.navigator.platform.match('Mac') ? event.metaKey : event.ctrlKey
+      if (!ctr /* CTRL | CMD */) return
+      if (key === 67) {
```

Comment:
not an showstopper, but for readability I would prefer something like `const isCopyAction = key === 67`